### PR TITLE
Added a note regarding the delegate_facts option.

### PR DESCRIPTION
**Pull request by Evertvz**
To avoid confusion if the user is not aware of the option delegate_facts.

##### SUMMARY

Added a line in the description informing the user about the option delegate_facts, in case he was not aware it existed.

##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/utilities/logic/include_vars.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 description:
     - Loads variables from a YAML/JSON files dynamically from within a file or from a directory recursively during task runtime. If loading a directory,
       the files are sorted alphabetically before being loaded.
+    - Please note that the use of 'delegate_facts: True' may be required depending on your use case when delegating the include.
```
Comment (nitzmahone) [inline lib/ansible/modules/utilities/logic/include_vars.py]: Agreed that this is a good change- should be a link to the docs, though, something like:
```
Please note that the use of :doc:`delegated facts </playbooks_delegation#delegated-facts>` may be required when delegating the include.
```

**Review by nitzmahone — DISMISSED** [review]

File: lib/ansible/modules/utilities/logic/include_vars.py (hunk 1 above)
Comment (Evertvz) [reply lib/ansible/modules/utilities/logic/include_vars.py]: I agree, that's indeed better. Changed.

File: lib/ansible/modules/utilities/logic/include_vars.py [hunk 2]
```diff
@@ -18,9 +18,11 @@
 module: include_vars
 short_description: Load variables from files, dynamically within a task
 description:
-  - Loads variables from a YAML/JSON files dynamically from within a file or from a directory recursively during task
-    runtime. If loading a directory, the files are sorted alphabetically before being loaded.
+  - Loads YAML/JSON variables dynamically from a file or from all files in a directory, recursively, during task runtime.
```
Comment (bcoca) [inline lib/ansible/modules/utilities/logic/include_vars.py]: why remove 'variables from a' ? that seems the important part of the phrase.

'all files' is not always true, as files are filtered by extension and/or the ignore option.

File: lib/ansible/modules/utilities/logic/include_vars.py [hunk 3]
```diff
@@ -18,9 +18,11 @@
...
+  - If loading a directory, the files are sorted alphabetically before being loaded.
   - This module is also supported for Windows targets.
+  - Please note that the use of L(delegate_facts,../user_guide/playbooks_delegate.html#delegated-facts) may be required,
+    depending on your use case, when delegating the include.
```
Comment (bcoca) [inline lib/ansible/modules/utilities/logic/include_vars.py]: I would rephrase this as 'to assign the variables to a different host than inventory_hostname you need to use both delegate_to and set delegate_facts: True'

**Review by bcoca — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/utilities/logic/include_vars.py (hunk 2 above)
Comment (acozine) [reply lib/ansible/modules/utilities/logic/include_vars.py]: I'll fix the "all files" bit. 
The old sentence had more `from`s than I thought were necessary - "from a YAML/JSON files . .  from within a file or from a directory". The new sentence is cleaner. But if it's not as clear, let's find a different way to say it.
